roles/tls-cert/README.md

```diff
@@ -26,21 +26,20 @@ If using this role to create and deploy a self-signed certificate then `openssl`
 Role Variables
 --------------
 
-| Variable                          | Required       | Default                   | Choices           | Comments                                         |
-|-----------------------------------|----------------|---------------------------|-------------------|--------------------------------------------------|
-| TLS_PRIVKEY_SRC_FILE              | no             |                           |                   | Path to private key on deployer system           |
-| TLS_CERT_SRC_FILE                 | no             |                           |                   | Path to certificate on deployer system           |
-| TLS_CACHAIN_SRC_FILE              | no             |                           |                   | Path to CA chain on deployer system              |
-| TLS_DEST_BASENAME                 | no             | provided cert CN*         |                   | Base filename of installed certificate           |
-| TLS_CREATE_SELFSIGNED             | no             | false                     | true, false       | Explicitly creates self-signed certificate       |
-| TLS_CERT_DEST_DIR                 | no             | (distro-specific)         |                   | Directory for certificates on target host        |
-| TLS_PRIVKEY_DEST_DIR              | no             | (distro-specific)         |                   | Directory for private keys on target host        |
-| TLS_SUBJECT_ALTERNATE_NAME        | depends        |                           |                   | The fully qualified domain name, only required   |
-|                                   |                |                           |                   | if creating a self-signed cert                   |
+| Variable                            | Required         | Default                     | Choices             | Comments                                                   |
```

Comment:
I would like to nuke this role, and perform a check that the user has installed if from galaxy